Dockerfile

```diff
@@ -28,13 +28,18 @@ RUN set -ex; \
 
 COPY . /app
 
+RUN mkdir -p /var/app/ && \
+  openssl ecparam -name prime256v1 -genkey -noout -out /var/app/vapid_private_key.pem && \
+  openssl ec -in /var/app/vapid_private_key.pem -pubout -out /var/app/vapid_public_key.pem
+
```

Comment:
I don't think this is a good idea since I believe subscriptions are tied to the key.


```suggestion
```